src/gl/specs/gl-api.spec

```diff
@@ -218,6 +218,10 @@ spec GL.API is
                           First : Int; Count : Size) with
      Static  => "glDrawArrays",
      Wrapper => "GL.Objects.Vertex_Arrays.Draw_Arrays";
+   procedure Draw_Arrays_Instanced (Mode  : Connection_Mode;
+     First : Int; Count : Size; Instances : Size) with
```

Comment:
Is there a particular reason `First` is of type `Int` and not `Size` here? It's `Size` in the public interface and does not seem to be able to get negative.